docs/TESTING.md

```diff
@@ -63,12 +49,24 @@ The tests are run against a live application, built in production mode. To make
 work consistently and do not depend on a specific schedule or realtime vehicle locations, we use
 [WireMock](http://wiremock.org/) to record and playback the V3 API responses.
 
-To run the tests, do the following:
+Prerequisites for running the tests:
+
+* Docker
+  * `brew cask install docker`
+  * Start Docker Desktop from the dock or Applications; this only has to be done
+    once, after which it will auto-start on login by default
+* Wiremock
+  * `brew cask install java --no-quarantine`
+    * This option is currently required on OS X 10.15+ due to Gatekeeper
+      changes. Ref: https://github.com/Homebrew/homebrew-cask/issues/70798
+  * `brew cask install wiremock-standalone`
+* Ensure `WIREMOCK_PATH` points to the Wiremock JAR file; with brew cask this
+  will be something like `/usr/local/Cellar/wiremock-standalone/<VERSION>/libexec/wiremock-standalone-<VERSION>.jar`
 
-* Make sure docker is [installed](https://docs.docker.com/docker-for-mac/install/) and running: we run the tests in docker to ensure we get consistent images across platforms.
-* Run `npm run backstop` which starts up wiremock, a mocked phoenix server, and when those are up, kicks off backstop testing in docker
+Once all the above are in place: `npm run backstop`
```

Comment:
I personally haven't gotten this macro to work. I've always had to do the three-bash approach listed here: https://github.com/mbta/wiki/blob/56ab636371b94604b62446e2f032909d16205dd2/website/testing/backstop.md#starting-the-phoenix-server-and-wiremock-separately

Reply:
Hm! I wasn't aware this separate doc existed. Should the docs in this repo be cut down to the basics, and link to that doc for full details? Or vice-versa: should we integrate those docs into this repo? Of course since Backstop is currently broken and not active on CI anyway, perhaps it's a moot point. I made these changes before I knew that.

Reply:
I defer to @ryan-mahoney and @phildarnowsky as to where this actually lives. I'm not sure our current policy on the generic WIKI vs. repo-specific documentation.

Reply:
@digitalcora since you are doing this now (and doing a great job!) I trust your judgment in terms of how to handle it. We don't have a policy

Reply:
I think it'd definitely be nice to get everything in one place; the existence of the `docs` directory in this repo (and absence of a link to the wiki) led me to think these were _the_ docs for the website project, when in reality many more docs existed in the wiki. Since the latter seems much more detailed and complete, I might advocate for merging this content into that, and having only the README with quick-start instructions here.

If that sounds good, I can either do that as a separate PR or revise this one.

Reply:
New thought: a factor in this decision might be which docs we'd like to be public-facing and which we don't, given the wiki is a private repo and this one is open-source.

Reply:
Although, there is the complication that the wiki is a private repo and this one is open-source, so we might want to keep some docs here if they would facilitate people outside the MBTA getting a fork of the website running.

Reply:
I am of two minds about this. On the one hand, I strongly dislike Github wikis (not least because I can never find the damn things), and I also dislike having multiple sources of documentation split up by no particular logic. On the other hand, moving everything from the wiki to the `docs` directory sounds like make-work, and it might be good to have a few things we hold back from the general public.

So let's keep these docs here for now, and I will add a ticket to investigate ways to automate moving wiki docs into the `docs` directory.

Reply:
@phildarnowsky Sorry, when I said "the wiki" I meant the `mbta/wiki` repository which Adamo linked in his first comment (which is not itself a GitHub Wiki™). I think we are all in agreement that docs should be a committed part of _some_ repo; the split right now is between the docs that live in `mbta/wiki` and the docs that live here in the `docs` directory.

Reply:
@digitalcora aha, my mistake. In that case let's continue not to mess with this for now and we'll discuss the question further at our next refinement.